config/v1/types_cluster_version.go

```diff
@@ -235,6 +235,11 @@ const (
 	// needed for the image streams to import the images they
 	// reference.
 	ClusterVersionCapabilityOpenShiftSamples ClusterVersionCapability = "openshift-samples"
+
+	// ClusterVersionCapabilityClusterBaremetalOperator manages the cluster
+	// baremetal operator which is responsible for running the metal3
+	// deployment.
+	ClusterVersionCapabilityClusterBaremetalOperator ClusterVersionCapability = "baremetal"
```

Comment:
nit: after [this thread][1], you moved the constant value to `baremetal`.  I'm fairly agnostic, but you might want to name the constant `ClusterVersionCapabilityBaremetal` to match.

[1]: https://github.com/openshift/api/pull/1125#discussion_r818158011